src/test/resources/schemas/describedBy.json

```diff
@@ -131,7 +133,7 @@
               "additionalProperties": false,
               "properties": {
                 "id": {
-                  "title": "lobid-organisations URI",
+                  "title": "lobid-organisations URI or external URI",
```

Comment:
Just curious: Where are non-lobid-organisations URIs used here and for `provider`and `modifiedBy`?